werkzeug/contrib/atom.py

```diff
@@ -21,12 +21,20 @@ def atom_feed(request):
     :copyright: (c) 2014 by the Werkzeug Team, see AUTHORS for more details.
     :license: BSD, see LICENSE for more details.
 """
+import warnings
 from datetime import datetime
-
 from werkzeug.utils import escape
 from werkzeug.wrappers import BaseResponse
+from werkzeug.contrib import WerkzeugContribDeprecationWarning
 from werkzeug._compat import implements_to_string, string_types
 
+warnings.warn(
+    'werkzeug.contrib.atom is deprecated as of version 0.15 and will'
+    ' be removed in version 1.0.',
```

Comment:
is there a known alternative we could suggest

Reply:
How about Jinja templating

Reply:
pure text template engines are very horrific for making xml output

Reply:
Quick search turned up [python-feedgen](https://github.com/lkiesow/python-feedgen). I don't think we need to recommend things in the warnings unless they're ours.